### ISSUE TITLE
macOS arm64 crash, resolved, missing db directories?

### ISSUE THREAD

**Issue by xjmzx** [lumehq/lume#163]
Had a few crashes after updating from 3.0.0 to 3.0.1. 

After updating successfully, seemed I changed something, maybe tried to remove one or two relays. Not sure if that was the cause or just high memory on syncing. Anyway, I couldn't get to start up without removing the folders:

```
/Users/my_user/Library/Application Support/nu.lume
/Users/my_user/Library/Application Support/nu.lume.Lume
```

After that it seemed to resolve. Not sure if it's necessary to have those db files in those folders. I just left them in the trash and now it's running fine again, there's no new folder repopulated in the Application Support directories. 

Mainly just wanted to ask for clarification on the directories and file structure on macOS, in case anything weird was going on.

I've attach some crash logs in case it's of any help (they may be similar.)

https://gist.github.com/xjmzx/0002e7cc8f017456ec56e6a89ef5670d
https://gist.github.com/xjmzx/479f886585d12cd81cba7a2118c630cc
https://gist.github.com/xjmzx/83101c259c859b95b48c89ff10a52f28
https://gist.github.com/xjmzx/2cbee136ffd8cc3e6289dd30e3feeb01
https://gist.github.com/xjmzx/dad413577c47ae95ca209afd2e2f7288
https://gist.github.com/xjmzx/004349ef64c30b2549c335823ee5b1fd

Apple  M1 Pro 16 GB  macOS 14.2.1 (23C71)

**Comment by reyamir** [comment]
Hi, thank for your detail report

The folder `/nu.lume` is old data path for Lume, and `/nu.lume.Lume` is new data path, the `.Lume` is required to build package for Flatpak Linux, so I've updated to support Flatpak

You can safely remove `/nu.lume` folder, Lume isn't use it anymore, only keep `/nu.lume.Lume`

Inside `/nu.lume.Lume`, Lume is storing these files:
- `sqlite db`: store cache and internal data
- `.tauri-plugin-theme`: store app theme config

So if you delete this folder, Lume is start fresh next time.

About your crash issue, look like there are bugs with relay connection, I will dig more into it.

**Comment by reyamir** [comment]
I'm also working on migrate all nostr codebase from JS to Rust via `rust-nostr` library to achive better performance and stability.

**Comment by xjmzx** [comment]
Thanks for the reply. Appreciate the information. I was just confused after I trashed them. Why they weren't populating in the same folder after restart. I ran `find -name nu.lume` and that shows the following directories.
``` 
/Users/x22/Library/WebKit/nu.lume
/Users/x22/Library/Caches/nu.lume
```

We seem to be running quite smoothly again. I left the `Sustainable for low network environment` toggled on my network can be unstable at best. 

Great to hear that there's more changes in the horizon, really enjoying the native app. Will see how it looks on Linux flatpak soon.
